frontend/packages/operator-lifecycle-manager/src/components/descriptors/status/index.tsx

```diff
@@ -64,6 +64,27 @@ const StatusConditions: React.FC<StatusCapabilityProps> = ({
   );
 };
 
+const ExternalLink: React.FC<StatusCapabilityProps> = ({
+  description,
+  fullPath,
+  label,
+  obj,
+  value,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <DetailsItem description={description} label={label} obj={obj} path={fullPath}>
+      {!_.isNil(value) ? (
+        <a href={value} rel="noopener noreferrer" target="_blank">
+          {value.replace(/https?:\/\//, '')}
+        </a>
```

Comment:
Reuse existing ExternalLink component. as it has specific styling. Also, don't remove scheme from URL. https://github.com/openshift/console/blob/67128172502d43de3ece30e7b11818006ec9d752/frontend/public/components/utils/link.tsx#L68

```suggestion
        <ExternalLink href={value}>
          {value}
        </a>
```